### PR TITLE
Style the hero teaser for the Soft Mod Webinar

### PR DESCRIPTION
**Pull request by LL782**
This completes the first draft of the styling and markup for the Soft Mod Webinar Hero Teaser.

Still to do, add the correct copy and translations.

The toggle is kept switched off for now.

@keith-smale @anguspaterson @Dan-Bird please review

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -43,6 +43,7 @@ validation-staus.json
 .CVS
 .idea
 node_modules
+vendor
```
Comment (keith-smale) [inline .gitignore]: Is PR 1746 required to be approved to enable this branch to go in?

Appreciate that we could just revert this if needed.

**Comment by keith-smale** [comment]
Looks good. 
We seem to lose half of Sandro's head as we go down the breakpoints to medium devices before it switches to mobile/small devices.

Is this potentially releasable?

File: .gitignore (hunk 1 above)
Comment (LL782) [reply .gitignore]: That PR has gone in now

**Comment by LL782** [comment]
> Looks good.
> We seem to lose half of Sandro's head as we go down the breakpoints to medium devices before it switches to mobile/small devices.
> 
> Is this potentially releasable?

This commit should address the image with the image: https://github.com/codurance/site/pull/1747/commits/49dfea5b33c9eaeb5215778a70faec787f886541

Regarding release, we still need the copy and link to be added.

**Review by keith-smale — APPROVED** [review]

**Pull request merged**